Content.Shared/Animal/WoolySystem.cs.cs

```diff
@@ -1,29 +1,29 @@
-using Content.Server.Animals.Components;
-using Content.Server.Chemistry.Containers.EntitySystems;
-using Content.Server.Nutrition;
+using Content.Shared.Chemistry.EntitySystems;
 using Content.Shared.Mobs.Systems;
+using Content.Shared.Nutrition;
 using Content.Shared.Nutrition.Components;
 using Content.Shared.Nutrition.EntitySystems;
 using Robust.Shared.Timing;
 
-namespace Content.Server.Animals.Systems;
+namespace Content.Shared.Animals;
 
 /// <summary>
-///     Gives ability to produce fiber reagents, produces endless if the 
-///     owner has no HungerComponent
+///     Gives ability to produce fiber reagents;
+///     produces endlessly if the owner has no HungerComponent.
 /// </summary>
 public sealed class WoolySystem : EntitySystem
 {
     [Dependency] private readonly HungerSystem _hunger = default!;
     [Dependency] private readonly IGameTiming _timing = default!;
     [Dependency] private readonly MobStateSystem _mobState = default!;
-    [Dependency] private readonly SolutionContainerSystem _solutionContainer = default!;
+    [Dependency] private readonly SharedSolutionContainerSystem _solutionContainer = default!;
 
     public override void Initialize()
     {
         base.Initialize();
 
         SubscribeLocalEvent<WoolyComponent, BeforeFullyEatenEvent>(OnBeforeFullyEaten);
+        Logger.Info("Subscribed to SharedBeforeFullyEatenEvent for SharedWoolyComponent");
     }
```

Comment:
This log should not exist nor should this need to be a message as subscriptions are entirely unordered.